src/dmd/dinifile.d

```diff
@@ -142,26 +142,21 @@ private bool writeToEnv(ref StringTable environment, char* nameEqValue)
  */
 void updateRealEnvironment(ref StringTable environment)
 {
-    static int envput(const(StringValue)* sv)
+    foreach (sv; environment)
```

Comment:
My guess would be to make this `ref sv`, just a guess though.

Reply:
That would be a good idea for later, perhaps. But for now StringTable.opApply() only supports const(StringValue)*